core/src/main/java/org/infinispan/remoting/transport/LocalModeAddress.java

```diff
@@ -0,0 +1,22 @@
+package org.infinispan.remoting.transport;
+
+/**
+ * @since 9.0
+ */
+public class LocalModeAddress implements Address {
+
+   public static final Address INSTANCE = new LocalModeAddress();
+
+   private LocalModeAddress() {
+   }
+
+   @Override
+   public String toString() {
+      return "Local Address";
+   }
+
+   @Override
+   public int compareTo(Address o) {
+      return 0;
```

Comment:
Yeah my guess is the compareTo is probably not used on this object, but just in case :)